packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -254,12 +319,31 @@ const ReflectionCard = (props: Props) => {
     !isComplete &&
     !isDemoRoute() &&
     (isHovering || !isDesktop)
+
+  const isGroupPhase = !isComplete && phaseType === 'group'
+  const hasNoGroup = !reflectionGroups.some((group) => group.reflections.length > 1)
+  const isMeetingNumberLessThanThree = meetingNumber < 3
+
+  const shouldAnimate = (() => {
+    if (isGroupPhase && hasNoGroup && isMeetingNumberLessThanThree) {
+      const firstReflection = reflectionGroups?.[0]?.reflections?.[0]
+      const isFirst = reflectionId === firstReflection?.id
+      const isNotEditing = !firstReflection?.isEditing
```

Comment:
-1 If I'm editing any card, then I don't want to be distracted. This should go into the next `.every` condition and then be renamed to `isNotInteracting` or sth. like that.